src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -601,12 +601,24 @@
     text: az sql midb list -g mygroup --mi myinstance
 """
 
+helps['sql midb list-deleted'] = """
+type: command
+short-summary: List restorable deleted managed databases.
+examples:
+  - name: List all restorable deleted managed databases on Managed Instance .
+    text: az sql midb list-deleted -g mygroup --mi myinstance
+"""
+
 helps['sql midb restore'] = """
 type: command
 short-summary: Restore a managed database.
 examples:
-  - name: Restore a managed database using Point in time restore
+  - name: Restore a live managed database using Point in time restore
     text: az sql midb restore -g mygroup --mi myinstance -n mymanageddb --dest-name targetmidb --time "2018-05-20T05:34:22"
+  - name: Restore a dropped managed database using Point in time restore
+    text: az sql midb restore -g mygroup --mi myinstance -n mymanageddb --dest-name targetmidb --time "2018-05-20T05:34:22" --deleted-time "2018-05-20T05:34:22"
+  - name: Restore a live managed database using Point in time restore from another instance
```

Comment:
nit: I think it might be more clear to say "Restore a live managed database from another instance using Point in time restore"

Reply:
Sure, will change.